lms/templates/problem.html

```diff
@@ -18,25 +18,37 @@ <h3 class="hd hd-2 problem-header">
     % if demand_hint_possible:
     <div class="problem-hint" aria-live="polite"></div>
     % endif
-    % if check_button:
-    <button class="check ${ check_button }" data-checking="${ check_button_checking }" data-value="${ check_button }"><span class="check-label">${ check_button }</span><span class="sr"> ${_("your answer")}</span></button>
-    % endif
+    <div class="problem-action-buttons-wrapper">
     % if demand_hint_possible:
-    <button class="hint-button" data-value="${_('Hint')}">${_('Hint')}</button>
-    % endif
-    % if reset_button:
-    <button class="reset" data-value="${_('Reset')}">${_('Reset')}<span class="sr"> ${_("your answer")}</span></button>
+    <span class="problem-action-button-wrapper">
+        <button class="hint-button btn btn-default btn-small" data-value="${_('Hint')}" title="${_('Hint')}"><span class="icon fa fa-question" aria-hidden="true"></span>${_('Hint')}</button>
```

Comment:
Why does there need to be a title given that the button has visible text? Forgive me if I've asked this previously.